tests/5.1/taskwait/test_task_nowait.c

```diff
@@ -0,0 +1,64 @@
+//--------------- test_taskwait_nowait.c---------------------------------//
+//
+// OpenMP API Version 5.1 Aug 2021
+//
+// Tests the behavior of taskwait when nowait is specified.
+// The behavior of this should be similar to that of using task construct.
+// Test was adapted to use the depend clauses which the nowait clause should ignore
+// It is important to note that all depenend tasks must be finished first.
+//-----------------------------------------------------------------------//
+
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+#include <unistd.h>
+
+#define N 1024
+
+int test_task_nowait(){
+	int errors = 0;
+	int test_scaler = 1;
+	int test_arr[N];
+	int sum = 0;
+	for (int i =0; i<N; i++){
+		test_arr[i] = 1;
+		sum += i;
+	}
+	#pragma omp parallel
+	#pragma omp single
+	{
+		#pragma omp task depend(out: test_scaler) shared(test_scaler)
```

Comment:
the dependence should be inout as it is modifying an existing value.

Reply:
inout and out are exactly the same under the hood / in the standard, so it is just an style issue.